src/components/ShortenedLink/ShortenedLink.jsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from "react";
+import { motion } from "framer-motion";
+import PropTypes from "prop-types";
+import {
+  ShortenedLinkContainer,
+  LinkToShorten,
+  ReadyLink,
+  IconContainer,
+} from "./ShortenedLink.styles";
+import { isLink } from "./ShortenedLink.utils";
+import Button from "../Button/Button";
+import { buttonSize } from "../Button/Button.styles";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faX } from "@fortawesome/free-solid-svg-icons";
+
+export default function ShortenedLink({ long, short, id, setLinkArr }) {
+  const [isCopied, setIsCopied] = useState(false);
+  const handleClick = () => {
+    navigator.clipboard
+      .writeText(short)
+      .then((res) => setIsCopied(true))
+      .catch((err) => console.log(err));
+    setTimeout(() => {
+      setIsCopied((prev) => !prev);
+    }, 5000);
+  };
+  const removeLink = () => {
```

Comment:
to w sumie można wyrzucić do pliku 'utils' dla tego komponentu. 
Funkcję wyżej - handleClick również, ale ze względu na to, że korzysta ona z hooka to konieczne byłoby stworzenie customowego hooka, ale tym zajmiemy się niedługo

Reply:
obie te funkcje korzystają z hooka - handleClick z setIsCopied, a removeLink z setLinkArr. No chyba, że coś mieszam :D